tests/test_openclhpp.cpp

```diff
@@ -1561,7 +1561,7 @@ static cl_mem clCreateImage3D_testCreateImage3D_1_1(
     TEST_ASSERT_EQUAL(16, image_depth);
     TEST_ASSERT_EQUAL(256, image_row_pitch);
     TEST_ASSERT_EQUAL(65536, image_slice_pitch);
-    TEST_ASSERT_EQUAL_PTR((void *) 0xdeadbeef, host_ptr);
+    TEST_ASSERT_EQUAL_PTR(nullptr, host_ptr);
```

Comment:
I don't think changing `0xdeadbeef` to `nullptr` is a good approach. I see the better solution is suggested in https://github.com/KhronosGroup/OpenCL-CLHPP/pull/223 by @Kerilk:
`    TEST_ASSERT_EQUAL_PTR((void *)(size_t)0xdeadbeef, host_ptr);
`
For this reason, this PR is also in conflict with the one mentioned above.

Reply:
Thanks 
You are right - changed back from nullptr to (void *)(size_t)0xdeadbeef
Other thing is that https://github.com/KhronosGroup/OpenCL-CLHPP/pull/223 
is not the same changes. This one and that are not conflicting- They are another fixes.